docs/documentation.md

```diff
@@ -0,0 +1,35 @@
+# Getting Started
```

Comment:
How about making this README.md so that it shows up in github as the default view?